components/src/main/java/uk/gov/hmrc/components/molecule/selectrow/SelectRowGroup.kt

```diff
@@ -143,9 +150,22 @@ class SelectRowGroup @JvmOverloads constructor(
             }
     }
 
+    private fun createDivider() = View(context).apply {
+        setBackgroundResource(R.drawable.components_divider)
+        layoutParams = LayoutParams(
+            ViewGroup.LayoutParams.MATCH_PARENT,
+            HMRC_SPACING_1.dpToPx().toInt()
+        ).apply {
+            marginStart = HMRC_SPACING_16.dpToPx().toInt()
+            marginEnd = HMRC_SPACING_16.dpToPx().toInt()
+        }
+    }
```

Comment:
Rather than creating a new divider here, could we use our existing Divider style to ensure it visually looks consistent with the other dividers (and make it easier to update if we need to change the divider for some reason). 

`ContextCompat.getDrawable(context, R.drawable.components_divider)`